docs/source/managed-federation/setup.mdx

```diff
@@ -71,33 +69,37 @@ const gateway = new ApolloGateway({
 
 This option is specific to _non_-managed federation, in which supergraph schema composition is performed via the Rover CLI.
 
-With managed federation, composition is instead performed by _GraphOS_, and the gateway regularly polls for an updated schema. This enables you to add, remove, and modify your subgraphs _without_ needing to restart your gateway.
+With managed federation, composition can instead be performed by _GraphOS_, where the gateway regularly polls for an updated schema. This enables you to add, remove, and modify your subgraphs _without_ needing to restart your gateway.
 
 Remove the `supergraphSdl` argument from your `ApolloGateway` constructor entirely:
 
 ```js
 const gateway = new ApolloGateway();
 ```
 
-> When running your gateway in an environment where outbound traffic to the internet is restricted, consult the [directions for configuring a proxy](/apollo-server/security/proxy-configuration) within Apollo Server.
+When running your gateway in an environment where outbound traffic to the internet is restricted, consult the [directions for configuring a proxy](/apollo-server/security/proxy-configuration) within Apollo Server.
 
 </ExpansionPanel>
 
-If you've already set up Apollo Federation _without_ connecting your router to GraphOS, you're probably passing the `--supergraph` (or `-s`) option to the Apollo Router's startup command:
+If you've already set up Apollo Federation _without_ connecting your router to GraphOS, you're probably passing the [`--supergraph` (or `-s`) option](/router/configuration/overview/#-s----supergraph) to the Apollo Router's startup command:
 
 ```sh
 ./router --config ./router.yaml --supergraph ./your-local-supergraph.graphql
 ```
 
-The `--supergraph` option is specific to _non_-managed federation, in which supergraph schema composition is performed via the Rover CLI and provided via a file path.
+The `--supergraph` option applies in scenarios where the supergraph schema for the router isn't retrieved by polling Apollo Uplink:
+
+* For enabling different production environments (such as blue-green deployments) to refer to a single graph variant, the supergraph schema for each production environment can be composed and retrieved by a workflow using Platform APIs, so that the routers of each environment use the supergraph schema corresponding to the environment. 
```

Comment:
I found this a little hard to follow and tried to break it up to make it more readable. That said, it seems a little contradictory to me. On the one hand, we seem to be saying that different production environments point to the same variant (and therefore the same schema?). On the other hand, we're saying each environments router can point to its (own?) corresponding supergraph schema? 

```suggestion
* **Enabling different production environments to refer to a single graph variant**, for example, for [blue-green](https://en.wikipedia.org/wiki/Blue%E2%80%93green_deployment): Workflows using the [Platform API](/graphos/platform-api/) compose and retrieve the supergraph schema for each production environment. This enables each environment's router to use the environment's supergraph schema. 
```